templates/artist/artist_application.html

```diff
@@ -5,7 +5,7 @@ <h3 class="extra-padding">Artist Application</h3>
     <div class="break">
         <hr>
     </div>
-    <p>Not sure how much to raise? Use our <a href="{% url 'funding' %}" target="_blank">Funding Calculator</a> to find out the best options.</p>
+    <h5>Want to learn more before applying? Check out our <strong><a href="{% url 'resources' %}" target=" _blank">Artist Resources</a></strong> page for more detils</h5>
```

Comment:
Couple nits on this line:
- Remove space before `_blank` for link target
- Use `<strong>` tags inside of `<a>` tags (reason for this is that semantically we are saying that the text should be strong instead of the link, though it is okay if we say that the link is on strong text rather than normal text)
- Typo: detils -> details

```suggestion
    <h5>Want to learn more before applying? Check out our <a href="{% url 'resources' %}" target="_blank"><strong>Artist Resources</strong></a> page for more details</h5>
```